ada_feeding/ada_feeding/helpers.py

```diff
@@ -92,6 +93,87 @@ def quat_between_vectors(vec_from: Vector3, vec_to: Vector3) -> Quaternion:
     return ret
 
 
+def add_update_static_tf(
```

Comment:
Name nit: `add_update_static_tf` seems confusing to me, why not just `set_static_tf`? And in the docstring you can document that if there is already a static tf for that key, it will be overridden (or you can make `override` a boolean flag)

Reply:
Okay on the name, no need for override boolean.